mix.exs

```diff
@@ -33,7 +33,7 @@ defmodule Dogma.Mixfile do
   defp deps do
     [
       # Test framework
-      {:shouldi, only: :test},
+      {:shouldi, only: ~w(dev test)a},
```

Comment:
You got it!

Do you think it's worth mentioning the coveralls task in the readme, perhaps [in the contributor section](https://github.com/lpil/dogma/tree/82d7e06dbdcab444c1a781ca796f1e1f5bb222a0#test-it)?